### PR TITLE
Wfp 2477 yorkshire and the humber onboarding

### PR DESCRIPTION
**Pull request by sambailey-moj**

### CODE REVIEW

**Review by wood-n — APPROVED** [review]
LGTM

File: src/main/resources/db/migration/V1_60__YORKSHIRE_AND_THE_HUMBER_ONBOARDING.sql [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS6', 'Hull Sentence Management Team 6', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55N55', 'Hull Sentence Management Team 10', 'N55HUL');
+
+INSERT INTO team (code, name, ldu_code) VALUES('N55AAK', 'Awaiting Allocation Kirklees', 'N55NKS');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS5', 'Kirklees Community Integration Team', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS1', 'Kirklees Sentence Management Team 1', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS2', 'Kirklees Sentence Management Team 2', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS3', 'Kirklees Sentence Management Team 3', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS4', 'Kirklees Sentence Management Team 4', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KUW', 'UPW- Kirklees Sentence Management Team 5', 'N55KIRL');
```
Comment (wood-n) [inline src/main/resources/db/migration/V1_60__YORKSHIRE_AND_THE_HUMBER_ONBOARDING.sql:105]: Does this want an extra space before the hyphen?

Comment (sambailey-moj) [reply src/main/resources/db/migration/V1_60__YORKSHIRE_AND_THE_HUMBER_ONBOARDING.sql]: good spot!

**Review by wood-n — APPROVED** [review]
LGTM

File: src/main/resources/db/migration/V1_60__YORKSHIRE_AND_THE_HUMBER_ONBOARDING.sql [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+INSERT INTO team (code, name, ldu_code) VALUES('N55A13', 'Awaiting Allocation North Yorkshire', 'N55NYNY');
+INSERT INTO team (code, name, ldu_code) VALUES('N55NYH', 'Harrogate - NY Sentence Management Team 2', 'N55NYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55NYN', 'Northallerton - NY Sentence Management Team 3', 'N55NYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55YNA', 'NY Community Integration Team East', 'N55NYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55YNC', 'NY Community Integration Team Selby', 'N55NYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55NYB', 'NY Community Integration Team West', 'N55NYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55NYP', 'NY PQiP Sentence Management 7', 'N55NYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55NYL', 'Selby - NY Sentence Management Team 5', 'N55NYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55NYK', 'Skipton - NY Sentence Management Team 4', 'N55NYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55YNU', 'UPW-NY Sentence Management Team 6', 'N55NYL');
```
Comment (wood-n) [inline src/main/resources/db/migration/V1_60__YORKSHIRE_AND_THE_HUMBER_ONBOARDING.sql:148]: No spaces on hyphen

**Review by wood-n — APPROVED** [review]
LGTM

**Pull request merged**